frontend/src/App.js

```diff
@@ -1,56 +1,56 @@
 import './App.css';
-import DestinationPageInformation from './DestinationPageInformation.js';
 import { BrowserRouter as Router, Route, Routes } from 'react-router-dom';
-import { useState } from 'react';
+import { useState, useEffect } from 'react';
+
 import Header from './components/Header.jsx';
 import Home from './pages/Home.jsx';
 import Login from './pages/Login.jsx';
 import Nopage from './pages/Nopage.jsx';
 import Cart from './pages/Cart.jsx';
-import Signup from "./pages/Signup.jsx";
+import Signup from './pages/Signup.jsx';
+import DestinationPageInformation from './DestinationPageInformation.js';
 
 function App() {
-  
-  const [id, setId] = useState("");
-  const [information, setInformation] = useState({
-    "photo_Url": "",
-    "title": "",
-    "location": "",
-    "tourism_type": "",
-    "description": "",
-    "event": "",
-    "price": "",
-    "rating": "",
-    "no_of_reviews": ""
-  });
+  const [pid, setPid] = useState(localStorage.getItem('pid') || '');
+  const [information, setInformation] = useState(
+    JSON.parse(localStorage.getItem('information')) || getDefaultInformation()
+  );
+
+  useEffect(() => {
+    localStorage.setItem('pid', pid);
+    localStorage.setItem('information', JSON.stringify(information));
```

Comment:
very bad way of stoarge staff try using a cookie
even my child (I do not have one) could sell data from such website

No problem in storage this not important information
but do not use for critical staff 

Sorry about that but
I hate hate hate websites that use localStorage
but it is ok for this not important info